tools/snp.sh

```diff
@@ -1112,138 +1100,136 @@ generate_snp_expected_measurement() {
     { >&2 echo -e "sev-snp-measure return value is empty"; return 1; }
   echo ${measurement}
 }
-
 attest_guest() {
-  local cpu_code_name=$(get_cpu_code_name)
+    local cpu_code_name=$(get_cpu_code_name)
 
-  # Install the sev-guest module
-  ssh_guest_command "sudo insmod /lib/modules/*/kernel/drivers/virt/coco/sev-guest/sev-guest.ko >/dev/null 2>&1 || true"
+    # Install the sev-guest module
+    ssh_guest_command "sudo insmod /lib/modules/*/kernel/drivers/virt/coco/sev-guest/sev-guest.ko >/dev/null 2>&1 || true"
 
-  # Request and display the snp attestation report with random data
-  if [ "$SVSM" = true ]; then
-	ssh_guest_command "sudo ./snpguest report attestation-report.bin request-data.txt --random"
-  else
-	ssh_guest_command "sudo ./$Home/snpguest/target/release/snpguest report attestation-report.bin request-data.txt --random --vmpl 3"
-  fi
-  ssh_guest_command "./$Home/snpguest/target/release/snpguest display report attestation-report.bin"
-
-  # Retrieve ark, ask, vcek (saved in ./certs)
-  ssh_guest_command "./$Home/snpguest/target/release/snpguest fetch ca pem ${cpu_code_name} ."
-  ssh_guest_command "./$Home/snpguest/target/release/snpguest fetch vcek pem ${cpu_code_name} . attestation-report.bin"
-
-  # Verifies that ARK, ASK and VCEK are all properly signed
-  ssh_guest_command "./$Home/snpguest/target/release/snpguest verify certs ."
+    # Request and display the snp attestation report with random data
+    if [ "$SVSM" = true ]; then
+      ssh_guest_command "sudo ./snpguest report attestation-report.bin request-data.txt --random"
+    else
+      ssh_guest_command "sudo ./snpguest report attestation-report.bin request-data.txt --random --vmpl 3"
+    fi
```

Comment:
Instead of re-writing the entire call, just create the string without the VMPL. Add your conditional logic below, appending the vmpl if SVSM is enabled...